packages/chart/src/RadialBar.ts

```diff
@@ -0,0 +1,167 @@
+            .text(d => d);
+        this.arcs
+            .selectAll('path')
+            .data(this.data())
+            .enter().append('path')
+            .attr('class', 'arc')
+            .on("mouseout.tooltip", this.tooltip.hide)
+            .on("mousemove.tooltip", this.tooltip.show)
+        this.svg.selectAll('.arc')
+            .style('fill', (d, i) => this._palette(i))
+            .transition()
+            .delay((d, i) => i * this.rowDelay())
+            .duration(this.tweenDuration())
+            .attrTween('d', (d, i) => {
+                let interpolate = d3interpolate(0, d[1]);
+                return t => this.arc(interpolate(t), i);
+            });
+        this.arcs
+            .selectAll('.arc-text')
+            .data(this.data())
+            .enter().append('text')
+            .attr('class', 'arc-text');
+        this.svg.selectAll('.arc-text')
+            .attr("x", -4)
+            .attr("y", (d, i) => -(this.getInnerRadius(i) + this.getOuterRadius(i)) / 2)
+            .attr("alignment-baseline", "middle")
+            .attr("text-anchor", "end")
+            .text(d => d[0])
+    }
+    getInnerRadius(index) {
+        return this.arcMinRadius() + (this.data().length - (index + 1)) * (this.arcWidth + this.arcPadding());
+    }
+    getOuterRadius(index) {
+        return this.getInnerRadius(index) + this.arcWidth;
+    }
+    rad2deg(angle) {
+        return angle * 180 / Math.PI;
+    }
+    //  INDChart
+    _palette;
+    fillColor: (row, column, value) => string;
+    strokeColor: (row, column, value) => string;
+    textColor: (row, column, value) => string;
+    click: (row, column, selected) => void;
+    dblclick: (row, column, selected) => void;
+    //  ITooltip
+    tooltip;
+    tooltipHTML: (_) => string;
+    tooltipFormat: (_) => string;
+    _selection;
+}
+RadialBar.prototype._class += " chart_RadialBar";
+RadialBar.prototype.implements(INDChart.prototype);
+RadialBar.prototype.implements(ITooltip.prototype);
+RadialBar.prototype.mixin(Utility.SimpleSelectionMixin);
+export interface RadialBar {
+    paletteID(): string;
+    paletteID(_: string): this;
+    radiusPadding(): number;
+    radiusPadding(_: number): this;
+    maxRange(): number;
+    maxRange(_: number): this;
+    maxDomainMultiplier(): number;
+    maxDomainMultiplier(_: number): this;
+    rowDelay(): number;
+    rowDelay(_: number): this;
+    tweenDuration(): number;
+    tweenDuration(_: number): this;
+    tickCount(): number;
+    tickCount(_: number): this;
+    arcPadding(): number;
+    arcPadding(_: number): this;
+    arcMinRadius(): number;
+    arcMinRadius(_: number): this;
+}
+RadialBar.prototype.publish("paletteID", "default", "set", "paletteID", RadialBar.prototype._palette.switch());
+RadialBar.prototype.publish("radiusPadding", 40, "number", "Padding between chart edge and container edge (px)", RadialBar.prototype._palette.switch());
```

Comment:
Should not have:  RadialBar.prototype._palette.switch()

Reply:
I'm not sure what you mean? This is declared just like the other chart widgets?

Reply:
*smack forehead* now I see what you mean...